ppgan/models/animeganv2_model.py

```diff
@@ -0,0 +1,210 @@
+import paddle
+from .base_model import BaseModel
+
+from .builder import MODELS
+from .generators.builder import build_generator
+from .discriminators.builder import build_discriminator
+from .losses import GANLoss
+from ..modules.caffevgg import CaffeVGG19
+from ..solver import build_optimizer
+from ..modules.init import init_weights
+from ..utils.filesystem import load
+from paddle import nn
```

Comment:
put this line in second line.  

system importing, than third-party lib importing, then locally importing